schedule/yast/maintenance/autoyast_migration_on_maintenance_updates.yaml

```diff
@@ -0,0 +1,26 @@
+---
+name: autoyast_migration_on_maintenance_updates
+description: >
+  Perform a migration between installed system with maintenance updates installed, and next release, eg sle15 sp3>sp4
```

Comment:
s/Perform/Performs/
s/maintenance updates/unreleased maintenance updates/ (I would like to do this distinction)

Reply:
Perform versus performs: both are correct, the symbol ":" introducing an explanation means we are continuing a sentence, and normally don't need an uppercase letter next, eg "(the object of the module is [to]) perfom" vs "(this) performs".
Then I agree for the next point :)

Reply:
makes sense, I have seen both ways, but never sure if was really ok, thanks!